zk-socials/social_validator/host/src/main.rs

```diff
@@ -60,7 +60,7 @@ mod tests {
             .unwrap();
 
         let prover = default_prover();
-        let receipt = prover.prove(env, SOCIAL_VALIDATOR_ELF).unwrap();
+        let receipt = prover.prove(env, SOCIAL_VALIDATOR_ELF).unwrap().receipt;
```

Comment:
based on this
<img width="876" alt="image" src="https://github.com/user-attachments/assets/57aaa529-924d-49be-8499-e223ba474e8e">